db/db_utils.py

```diff
@@ -21,7 +21,10 @@ def __init__(self, url, metadata=None):
             self.metadata = metadata
 
 
-class DbUtils(object):
+class DbInstance(object):
+
+    db = None
+
     def _establish_db_connection(self):
         # Establish connection
         connectionString = 'postgres://' + base64.b64decode(self._username) + ':' + base64.b64decode(self._password) + '@' + base64.b64decode(self._hostName) + '/' + self._dbName
```

Comment:
Just realized this is just building ontop of the existing db file, I just make the above a seperate task